whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/dao/Nrtm4ClientMirrorRepository.java

```diff
@@ -0,0 +1,83 @@
+package net.ripe.db.nrtm4.client.dao;
+
+import net.ripe.db.whois.common.DateTimeProvider;
+import net.ripe.db.whois.common.dao.jdbc.JdbcRpslObjectOperations;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.sql.Statement;
+import java.util.List;
+
+@Repository
+public class Nrtm4ClientMirrorRepository {
```

Comment:
You need to have conditional here as well , else it will look for that datasource in Prod

Reply:
added